### ISSUE TITLE
Chembl

### ISSUE THREAD

**Issue by eduardszoecs** [ropensci/webchem#109]
https://www.ebi.ac.uk/chembl/api/data/docs
https://www.ebi.ac.uk/chembl/ws

**Comment by Aariq** [comment]
### Feasibility
Has a well documented REST API. Has a [python client](https://github.com/chembl/chembl_webresource_client) as well, for inspiration

There are probably too many possible requests to implement in webchem (there could easily be a stand-alone chembl package), but maybe there are a few things that aren't covered by other databases that we'd want to implement.

### Scope

- converts among common identifiers (smiles, inchikey, inchi)
- converts to and from molfiles
- converts *images* to molfiles
- produces images from identifiers and molfiles
- bioactivity, assay, and target activity
- target predictions
- cross reference ID translations to other databases

### Overlap

- Seems like the database leans more biological than perhaps some other databases already in webchem
- converting *from* images is certainly not covered, but probably not within scope
- I don't think target activity and predicted targets is already covered.
- overall seems like a good addition.

**Comment by Aariq** [comment]
Note: the translator functions might be worth it alone as an alternative to CTS which seems less reliable.

**Comment by stitam** [comment]
Definitely, thanks @jranke for flagging this. Access to ChEMBL has been implemented.
